### ISSUE TITLE
FAILED Compilation failure in tests

### ISSUE THREAD

**Issue by johanandren** [akka/akka#22243]
@jrudolph said he had seen it and suspected race condition in multi-jvm or something like that.

Manifests for example like this:

https://jenkins.akka.io:8498/job/akka-multi-node/1923/consoleFull

```
[error] missing or invalid dependency detected while loading class file 'LeveldbIdMapping.class'.
[error] Could not access term iq80 in package org,
[error] because it (or its dependencies) are missing. Check your build definition for
[error] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[error] A full rebuild may help if 'LeveldbIdMapping.class' was compiled against an incompatible version of org.
[error] missing or invalid dependency detected while loading class file 'LeveldbStore.class'.
[error] Could not access term iq80 in package org,
[error] because it (or its dependencies) are missing. Check your build definition for
[error] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[error] A full rebuild may help if 'LeveldbStore.class' was compiled against an incompatible version of org.
[error] two errors found
```

**Comment by johanandren** [comment]
https://jenkins.akka.io:8498/job/akka-multi-node/1924/consoleFull
https://jenkins.akka.io:8498/job/akka-multi-node/1925/consoleFull
https://jenkins.akka.io:8498/job/akka-multi-node/1926/consoleFull

**Comment by patriknw** [comment]
I fixed this the day before the 2.5-M1 release and those tests looks old, so I think this is already fixed. Reopen if you see it again.
